agent/runner/jobs/mongodb_restore_job.go

```diff
@@ -106,6 +116,7 @@ func (j *MongoDBRestoreJob) Run(ctx context.Context, send Send) error {
 		return errors.WithStack(err)
 	}
 
+	defer j.restarter.RestartAgents()
```

Comment:
Don't we need to restart agents only if restoring has been done successfully?

Reply:
I guess unsuccessful restore may cause problems as well. Better to restart.